jvb/rootfs/etc/cont-init.d/10-config

```diff
@@ -19,4 +19,9 @@ if [[ ! -f /config/logging.properties ]]; then
     cp /defaults/logging.properties /config
 fi
 
+if [[ ! -z $JVB_JVM_MAX_MEM ]]; then 
+    echo -n "# The amount of memory that the java process can consume.\n# Use the format that is in use for java's -Xmx switch# The default is 3072m\n# Only effective on linux, 64 bit \nVIDEOBRIDGE_MAX_MEMORY=$JVB_JVM_MAX_MEM
+\n# Uncomment the next line to enable the remote debugging of the video bridge \n# VIDEOBRIDGE_DEBUG_OPTIONS=\"-Xdebug -Xrunjdwp:transport=dt_socket,server=y,suspend=n,address=8000\"" > /usr/share/jitsi-videobridge/libs/viedobridge.rcfi
```

Comment:
please drop this as it'ss unrelated to the change

Reply:
Sorry Saghul but I don't understand your comment.
The env var and the creation of the rc file are the change

Reply:
This part: # Uncomment the next line to enable the remote debugging ...

Reply:
Sur, it's removed